sdk/include/vcruntime/immintrin.h

```diff
@@ -164,6 +208,149 @@ __INTRIN_INLINE int _rdseed64_step(unsigned __int64* random_val)
 }
 #endif // __x86_64__
 
+__INTRIN_INLINE void _fxsave(void *__P)
+{
+#if 0 // Needs newer GCC
+    __builtin_ia32_fxsave(__P);
+#else
+    asm volatile ("fxsave (%0)" : : "r"(__P));
```

Comment:
```suggestion
    __asm__ __volatile__("fxsave (%0)" : : "r"(__P));
```
Also, `__volatile__` (as done above), or, `__volatile` as was used in the new definition you've introduced below?